client/go/cmd/vespa/main.go

```diff
@@ -6,8 +6,11 @@ package main
 
 import (
 	"github.com/vespa-engine/vespa/client/go/cmd"
+	"os"
 )
 
 func main() {
-	cmd.Execute()
+	if cmd.Execute() != nil {
```

Comment:
Nit: `if err := ...; err != nil {` is a common idiom to indicate that the returned value is in fact an error.

Reply:
Too bad `cobra.Command{Run: ...}` does not return an error, because then we could pass all errors up to here.

Reply:
Which kind of errors are masked by cobra.Command?

Reply:
Fixed